packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/types.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity } from '@dcl/ecs'
+
+export interface Props {
+  entity: Entity
+}
+
+export enum Actions {
+  PLAY_ANIMATION = 'play_animation',
+  ACTIVATE = 'activate',
+  DEACTIVATE = 'deactivate',
+  GO_TO_END = 'go_to_end',
+  GO_TO_START = 'go_to_start'
```

Comment:
what are the other besides `play_animation`?

Reply:
Added for testing purposes, removing them